tests/Doctrine/Tests/DBAL/ConnectionTest.php

```diff
@@ -403,13 +403,12 @@ public function testFetchColumn()
         $this->assertSame($result, $conn->fetchColumn($statement, $params, $column, $types));
     }
 
-    public function testConnectionIsClosed()
+    public function testGetPlatformVersionDoesNotFailOnBadConnection()
     {
-        $this->_conn->close();
-
-        $this->setExpectedException('Doctrine\\DBAL\\Exception\\DriverException');
-
-        $this->_conn->quoteIdentifier('Bug');
+        // the connection uses bogus connection requirements, so connecting
+        // *will* fail. We should be able to call this without connection
+        // and causing that exception
+        $this->_conn->getDatabasePlatform();
```

Comment:
Cool, I would like to hear what @deeky666 thinks then :).

I'm here because the exception has wide-reaching consequences that are affecting Symfony users - e.g. doctrine/DoctrineBundle#351 and symfony/symfony-standard#774. 

Thanks for the attention!